courses/templates/course_detail.html

```diff
@@ -3,34 +3,43 @@
 
 {% block title %}MIT xPro | {{ course.title }}{% endblock %}
 
+{% block extrahead %}
+<style>
+  .header-block {
+    background-image: url('{{ course.background_image_mobile_url }}');
+  }
+
+  @media screen and (min-width: 1024px) { 
+    .header-block {
+      background-image: url('{{ course.background_image_url }}'); 
+    }
+  }
+</style>
+{% endblock %}
+
 {% block content %}
-<div class="course-detail">
-  <h1>{{ course.title }}</h1>
-  <h3>{{ course.description }}</h3>
-  <h3>Course Runs:</h3>
-  {% for course_run in course.courserun_set.all %}
-    <div style="background-color: {% cycle "#eee" "#fff" %}; padding: 10px;">
-      <h4 style="margin: 0 0 10px 0; padding: 0;">
-        {% if course_run.courseware_url %}
-            &#11016;
-            <a
-              href="{{ course_run.courseware_url }}"
-              target="_blank"
-              rel="noopener noreferrer"
-            >
-              {{ course_run.title }}
-            </a>
-        {% else %}
-          {{ course_run.title }}
+<div class="header-block">
+  <div class="container header-container">
+    {% if course.program %}
+    <span class="program-position">Course {{ course.position_in_program }} of {{ course.program.num_courses }} in the {{ course.program.title }}</span>
+    {% endif %}
+    <div class="row">
+      <div class="col-lg-7">
+        <div class="caption">
+          <h1>{{ course.display_title|safe }}</h1>
+          <h2>{{ course.description|safe }}</h2>
```

Comment:
I think we need a new field for this. The "details" part of [this course catalog design](https://impactbnd.invisionapp.com/share/4TQEESTEYMZ#/screens/354198017_Course_Catalog_V3) is where we currently output the `description` field. This subheading is not likely to be the same text, and it definitely should just be plain text, not rich text. We can call the field something like `subheading`

Reply:
Created `subhead` field for this on `ProductPage`